README.md

```diff
@@ -36,44 +35,64 @@ Follow the rest of the setup instructions (providing the environment credentials
 
 **Important**
+
 1. Heroku servers are currently unavailable because the free services we are using are not able to cope with BridgeInTech architecture. Detailed explanation about this can be found [here](https://medium.com/anitab-org-open-source/why-you-cant-use-heroku-free-dynos-in-multiple-servers-dependent-application-fa1cdd9e9e07?source=friends_link&sk=281bb76050f052c03c6772e9be275f67)
 
 2. Even if we found the way to make Heroku servers work, please be aware that there are **drawbacks** if you chose to run the BIT and MS backend servers remotely (**option 2**):
-* There will be a **significant delay** from the time when the request is made on the BIT web GUI to the time the response is received. This is because BIT Heroku makes calls to MS-for-BIT Heroku backend server for MS related functionalities. Both of these Heroku servers are running on [free `dynos`](https://www.heroku.com/dynos) that have limited capacity. When you run both BIT and MS-for-BIT servers locally, you will not have this time lag issue.
-* The BIT Heroku backend server will only carry the latest code that have been approved and merged to the develop branch. This means, if you are working on a PR or testing an open PR, this **Option 2** (running remote BIT Heroku server) **MUST NOT BE USED**.     
+
+- There will be a **significant delay** from the time when the request is made on the BIT web GUI to the time the response is received. This is because BIT Heroku makes calls to MS-for-BIT Heroku backend server for MS related functionalities. Both of these Heroku servers are running on [free `dynos`](https://www.heroku.com/dynos) that have limited capacity. When you run both BIT and MS-for-BIT servers locally, you will not have this time lag issue.
+- The BIT Heroku backend server will only carry the latest code that have been approved and merged to the develop branch. This means, if you are working on a PR or testing an open PR, this **Option 2** (running remote BIT Heroku server) **MUST NOT BE USED**.
 
 ## Run the app
 
 Now that you have set the MS backend, BIT backend and frontend servers, you can run BIT React application by following the steps below:
 
 1. install dependencies by running the command below
+
 ```
 $ npm install
 ```
 
 2. run the app using the command below
+
 ```
 $ npm start
 ```
 
 3. go to the browser on http://localhost:3000/
 
-
 **NOTE**:If you have chosen to use local BIT and MS backend servers, make sure you run both of them as well as the frontend server. If you chose the Heroku BIT backeend server, then go to `config.js` file and uncomment the Heroku server url and comment out the localhost url instead (see screenshot below).
 
 <img width="1354" alt="Screen Shot 2020-07-22 at 11 03 19 pm" src="https://user-images.githubusercontent.com/29667122/88179650-931fea80-cc6f-11ea-818c-8e412432c926.png">
 
+## UI Screenshots
+
+**Home Section (About and Benefit)**
+
+<img width="400" src="screenshots/about.png">
```

Comment:
@amlannandy , instead of uploading the images here directly from your local machine (they will be shown as broken links when we merge the issue to develop branch since the remote repo doesn't have access to contributor's local machine). please use the url link to images (see line 73 for example). Let me know if you need help on this.